packages/common/client-utils/api-report/client-utils.legacy.beta.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-export { EventEmitter }
```

Comment:
Is it correct that this disappeared from the legacy.beta and legacy.public API reports?